appengine-java8/pubsub/src/main/java/com/example/appengine/pubsub/PubSubAuthenticatedPush.java

```diff
@@ -20,30 +20,46 @@
 import com.google.api.client.googleapis.auth.oauth2.GoogleIdTokenVerifier;
 import com.google.api.client.http.javanet.NetHttpTransport;
 import com.google.api.client.json.jackson2.JacksonFactory;
+import com.google.gson.Gson;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonParser;
 import java.io.IOException;
+import java.util.Base64;
 import java.util.Collections;
+import java.util.stream.Collectors;
 import javax.servlet.ServletException;
 import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+// [START gae_flex_pubsub_auth_push]
```

Comment:
This is for [GAE Standard](https://github.com/GoogleCloudPlatform/java-docs-samples/tree/1c394688a61815ed14d74b23ce1f49ae77be3517/appengine-java8/pubsub) actually. Please use `gae_standard_pubsub_auth_push`.

Reply:
done